### ISSUE TITLE
Don't include 22GB file by default in the workspace on this repo

### ISSUE THREAD

**Issue by meysholdt** [gitpod-io/gitpod#4810]
Workspaces start and stop rather slowly on this repository. 

I believe this is due to this large file:
```
312M    ./gitpod/components/ee/agent-smith/scripts/_output/rootfs.tar.gz
385M    ./gitpod/node_modules/@babel
555M    ./go/pkg/mod/google.golang.org
1.5G    ./go
1.5G    ./go/pkg
1.5G    ./go/pkg/mod
1.8G    ./gitpod/node_modules
22G     ./gitpod/components
22G     ./gitpod/components/ee
22G     ./gitpod/components/ee/agent-smith
22G     ./gitpod/components/ee/agent-smith/scripts
22G     ./gitpod/components/ee/agent-smith/scripts/_output
22G     ./gitpod/components/ee/agent-smith/scripts/_output/bionic-server-cloudimg-amd64.img
24G     ./gitpod
25G     .
```

(to reproduce, run `du -hxa -d 8 | sort -h` in `/workspace`)

Proposal: 
Don't include `bionic-server-cloudimg-amd64.img` into prebuilds of this repo because (I assume) most people that work on this repo don't use it. Maybe do the same with `rootfs.tar.gz`. Instead, have a script that can download the file on-demand.

CC @fntlnz

**Comment by meysholdt** [comment]
I see a PR addressed this yesterday: https://github.com/gitpod-io/gitpod/pull/4766

However, in workspaces that I start today I still see the 22GB file.

**Comment by geropl** [comment]
> However, in workspaces that I start today I still see the 22GB file.

Maybe based on an old prebuild? The trigger is [here](https://github.com/gitpod-io/gitpod/blob/main/.gitpod.yml#L36).

**Comment by csweichel** [comment]
Closing because this is fixed in #4766 - if we still see this issue in a few days, please reopen
